src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+-- }}} credentials.users configuration
+
+-- }}} credentials configuration
+
+-- {{{ database configuration
+
+M['database'] = format_text([[
+    The `database` section defines database-specific configuration parameters,
+    such as an instance's read-write mode or transaction isolation level.
+]])
+
+M['database.hot_standby'] = format_text([[
+    Whether to start the server in the hot standby mode. This mode can be used
+    to provide failover without replication.
+
+    Note: `database.hot_standby` has no effect:
+
+    - If `wal.mode` is set to none.
+    - If `wal.dir_rescan_delay` is set to a large value on macOS or FreeBSD.
+    On these platforms, the hot standby mode is designed so that the loop
+    repeats every `wal.dir_rescan_delay` seconds.
+    - For spaces created with engine set to `vinyl`.
+]])
+
+M['database.instance_uuid'] = format_text([[
+    An instance UUID.
+
+    By default, instance UUIDs are generated automatically.
+    `database.instance_uuid` can be used to specify an instance identifier
+    manually.
+
+    UUIDs should follow these rules:
+
+    - The values must be true unique identifiers, not shared by other instances
+    or replica sets within the common infrastructure.
+    - The values must be used consistently, not changed after the initial setup.
+    The initial values are stored in snapshot files and are checked whenever
+    the system is restarted.
+    - The values must comply with RFC 4122. The nil UUID is not allowed.
+]])
+
+M['database.mode'] = format_text([[
+    An instance's operating mode. This option is in effect if
+    `replication.failover` is set to `off`.
+
+    The following modes are available:
+
+    - `rw`: an instance is in read-write mode.
+    - `ro`: an instance is in read-only mode.
+
+    If not specified explicitly, the default value depends on the number of
+    instances in a replica set. For a single instance, the `rw` mode is used,
+    while for multiple instances, the `ro` mode is used.
+]])
+
+M['database.replicaset_uuid'] = format_text([[
+    A replica set UUID.
+
+    By default, replica set UUIDs are generated automatically.
+    `database.replicaset_uuid` can be used to specify a replica set identifier
+    manually.
+]])
+
+M['database.txn_isolation'] = 'A transaction isolation level.'
+
+M['database.txn_timeout'] = format_text([[
+    A timeout (in seconds) after which the transaction is rolled back.
+]])
+
+M['database.use_mvcc_engine'] = 'Whether the transactional manager is enabled.'
+
+-- }}} database configuration
+
+-- {{{ failover configuration
+
+M['failover'] = format_text([[
+    The `failover` section defines parameters related to a supervised failover.
+]])
+
+M['failover.call_timeout'] = format_text([[
+    A call timeout (in seconds) for monitoring and failover
+    requests to an instance.
+]])
+
+M['failover.connect_timeout'] = format_text([[
+    A connection timeout (in seconds) for monitoring and failover
+    requests to an instance.
+]])
```

Comment:
> A connection timeout for requests

Sounds unusual. I would say 'a timeout for connecting' or just 'a connection timeout'.

Reply:
```lua
I['failover.connect_timeout'] = format_text([[
    A connection timeout (in seconds).
]])
```

Reply:
Hm. To be honest, I liked that we say that it is for connecitions used by monitoring and autofailover components.

The failover coordinator has various connections: some of them are to etcd (using http), some to tarantool instances (using iproto). And this timeout affects ones that use iproto.

If we don't say it, it may create a false impression that this timeout affects all the connections.

Maybe it should, hm. But it doesn't do for now.

----

Funny, the original wording is mine :) (from https://github.com/tarantool/enterprise_doc/issues/253)

Reply:
Let's leave it like this
```lua
I['failover.connect_timeout'] = format_text([[
    A connection timeout (in seconds) for connecitions
    used by monitoring and autofailover components.
]])
```

Reply:
This way seems good for me, thanks!

Typo: connecitions -> connections

Please, also mention the components in `failover.call_timeout`.

Reply:
Done.